apps/swiss/sh/util.c

```diff
@@ -1112,7 +1112,7 @@ Return Value:
 
 {
```

Comment:
(I couldn't seem to comment on the commit message itself so I'll comment here)

Please include a description of the change and why it's needed in the commit message. Take a look at "git log" for examples. I see you included the information as a comment in the pull request, but once these changes are integrated into the tree that information will be lost to anyone looking to understand this change.